hls-plugin-api/src/Ide/Types.hs

```diff
@@ -568,11 +568,27 @@ instance PluginRequestMethod Method_CodeActionResolve where
     -- See Note [Resolve in PluginHandlers].
     combineResponses _ _ _ _ (x :| _) = x
 
+combineDefinitions :: NonEmpty (Definition |? ([DefinitionLink] |? Null)) -> (Definition |? ([DefinitionLink] |? Null))
```

Comment:
Is there specific reasoning for how this is defined? If yes, a comment would be nice.

Reply:
I removed this now as this was already done by https://github.com/haskell/haskell-language-server/pull/3846